presto-hive/src/main/java/io/prestosql/plugin/hive/HiveConfig.java

```diff
@@ -977,4 +980,31 @@ public HiveConfig setProjectionPushdownEnabled(boolean projectionPushdownEnabled
         this.projectionPushdownEnabled = projectionPushdownEnabled;
         return this;
     }
+
+    public boolean isRedirectingToIcebergEnabled()
+    {
+        return redirectingToIcebergEnabled;
+    }
+
+    @Config("hive.redirecting-to-iceberg-enabled")
```

Comment:
Let's name these
* `hive.redirect-to-iceberg.enabled`
* `hive.redirect-to-iceberg.catalog`